cpp/ql/lib/semmle/code/cpp/ir/implementation/raw/internal/TranslatedExpr.qll

```diff
@@ -3437,8 +3460,7 @@ class TranslatedVarArg extends TranslatedNonConstantExpr {
 
   final override Instruction getInstructionSuccessorInternal(InstructionTag tag, EdgeKind kind) {
     tag = VarArgsVAListLoadTag() and
-    kind instanceof GotoEdge and
-    result = this.getInstruction(VarArgsArgAddressTag())
+    (kind instanceof GotoEdge and result = this.getInstruction(VarArgsArgAddressTag()))
```

Comment:
This parenthesis here is also superfluous now

Reply:
So much for the formatting tool fixing that automatically.